### DISCUSSION TITLE
Resize integrations

### DISCUSSION THREAD

**Discussion by mont5piques** [outline/outline#1988] [Ideas]
In some cases, we may need to resize integrations height to fit the content (aitable for example).

I don't know how this could be achieved, maybe using a mini-toolbar like image one which will offer 3 sizes buttons (small, medium and large).

Thanks

**Comment by gusjux** [comment]
Yes, it would be very helpful! For example I want to embed a google calendar or sheet with 600x600px size.

**Comment by KarloDerEchte** [comment]
Yes we need that!

**Comment by dreis06** [comment]
Yes, that would be very helpful. I am trying to embed some Grist view and need more than 400px height for that

**Reply by tommoor** [reply]
Another one blocked by #3000 – just not possible yet due to using Markdown as the serialization format
